tests/integration/test_query.py

```diff
@@ -165,7 +165,7 @@ def index(sample_data, redis_url):
                     "attrs": {
                         "dims": 3,
                         "distance_metric": "cosine",
-                        "algorithm": "flat",
```

Comment:
@abrookins, catching up to this, didn't see a test for it (but may have missed it) what happens if algorithm is `flat` and you set ef_runtime? Does it just do nothing